lib/rails_admin_import/action.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_admin/config/actions'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.